wgpu-core/src/conv.rs

```diff
@@ -594,6 +594,7 @@ pub(crate) fn map_buffer_state(usage: resource::BufferUse) -> hal::buffer::State
         access |= A::SHADER_READ;
     }
     if usage.contains(W::STORAGE_STORE) {
+        access |= A::SHADER_READ;
```

Comment:
let's write it as `A::SHADER_READ | A::SHADER_WRITE`, consistently with the other arm

Reply:
Missed it. Fixed!